docs/date-time-picker.md

```diff
@@ -1,7 +1,7 @@
 <a name="UiDateTimePicker"></a>
 
 ## UiDateTimePicker ⇐ <code>DateTimePickerBase</code>
-A widgets to edit dates and times.
+A widgets to set a date and time of day.
```

Comment:
Why "of a day"? Seems strange...

Reply:
To make the distinction clearer between "date" and "time". (First changed this in time-picker and then here for constistency).

Can change it back if you like.

Reply:
> Can change it back if you like.

I'm unsure... leave it as it is by now (I mean let's keep "of day")

Reply:
Too late. (I'm no native speaker either).